### DISCUSSION TITLE
BPM seems to reset upon exiting back to main menu when failing a song

### DISCUSSION THREAD

**Discussion by eggooooo** [ppy/osu#20192] [Q&A] [answered]
When exiting a song, objects that would usually pulse to the bpm seem to just be stuck (e.g osu! logo, menu icons etc)

https://user-images.githubusercontent.com/64386789/189114176-a47717d8-0842-4fef-8c7a-b51095cd5b5a.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20193
